src/Environment.php

```diff
@@ -376,15 +376,7 @@ public function load($name)
      *
      * @internal
      */
-    public function loadTemplate($name, $index = null)
```

Comment:
Shouldn't this be deprecated first in the previous version? It's a BC break without previous warning.

Reply:
NVM, I just saw that it's marked as `@internal` method